CHANGELOG.md

```diff
@@ -1,3 +1,172 @@
+# 1.9.0 (September 2025)
```

Comment:
Can we include a link to https://www.jetbrains.com/help/kotlin-multiplatform-dev/whats-new-compose-190.html somewhere?

Would be good to have it for all *.0 releases.